cpp/benchmarks/tpch/utils.hpp

```diff
@@ -28,17 +30,104 @@
 #include <cudf/transform.hpp>
 #include <cudf/unary.hpp>
 
-#include <rmm/cuda_device.hpp>
+#include <rmm/device_uvector.hpp>
+#include <rmm/mr/device/cuda_async_memory_resource.hpp>
 #include <rmm/mr/device/cuda_memory_resource.hpp>
 #include <rmm/mr/device/device_memory_resource.hpp>
 #include <rmm/mr/device/managed_memory_resource.hpp>
 #include <rmm/mr/device/owning_wrapper.hpp>
 #include <rmm/mr/device/pool_memory_resource.hpp>
+#include <rmm/mr/device/prefetch_resource_adaptor.hpp>
+#include <rmm/mr/device/statistics_resource_adaptor.hpp>
 
+#include <cstdlib>
 #include <ctime>
 
+namespace {
+const std::vector<std::string> ORDERS   = {"o_orderkey",
```

Comment:
We use east const in libcudf. (Reference [libcudf C++ Guidelines](https://github.com/rapidsai/cudf/blob/branch-24.10/cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md#c-guidelines))
```suggestion
std::vector<std::string> const ORDERS   = {"o_orderkey",
```